package.json

```diff
@@ -18,7 +19,8 @@
     "heroku-client": "^1.9.1",
     "isomorphic-fetch": "^2.0.0",
     "origami-build-tools": "^2.13.0",
-    "semver": "^4.3.1"
+    "semver": "^4.3.1",
+    "fastly": "git+https://github.com/Financial-Times/fastly.git#v2.5.0"
```

Comment:
heroku-client isn't a dependency of haikro, haikro just talks to the REST api directly but that sounds good!